non-atomic-action-interpreter-module/README.md

```diff
@@ -0,0 +1,135 @@
+# Non-Atomic Action Interpreter Module
+
+This module is part of the ostis-ps-lib and is responsible for interpreting non-atomic actions within the OSTIS.
+
+## Using module as an Extension for sc-machine
+
+The non-atomic action interpreter Module can be used as an extension to the sc-machine. Follow these steps to integrate it:
+
+### 1. Download sc-machine Artifacts
+
+   - Download pre-built artifacts of sc-machine from [GitHub Releases](https://github.com/ostis-ai/sc-machine/releases) and extract them to a location of your choice.
+   - Alternatively, you can build sc-machine from sources and use the resulting artifacts.
+   - For more details on installing and using sc-machine artifacts, refer to the [sc-machine quick start guide](https://ostis-ai.github.io/sc-machine/quick_start/).
+  
+### 2. Download Non-Atomic Action Interpreter Module Artifacts
+
+   - Download pre-built artifacts of Non-Atomic Action Interpreter Module from [GitHub Releases](https://github.com/ostis-ai/ostis-ps-lib/releases) and extract them to a location of your choice.
+   - Alternatively, you can build Non-Atomic Action Interpreter Module from sources and use the resulting artifacts (see Developing Module section).
+
+### 3. Specify Extension Paths
+
+   - When running the sc-machine binary, specify the path to the `lib/extensions` directory from the extracted Non-Atomic Action Interpreter Module folder:
```

Comment:
no extensions in directory
![image](https://github.com/user-attachments/assets/f18750d1-6c1f-497a-821c-6db06e3dd580)
![image](https://github.com/user-attachments/assets/a6f7c562-f6a1-4a28-8e2c-8f1e4bf48158)